tests/create_test_dataset.py

```diff
@@ -0,0 +1,145 @@
+                              area_average=False, 
+                              name='DataSelector00101', 
+                              resolution=resolution, 
+                              scenario=scenarios, 
+                              time_slice=(year_start, year_end), 
+                              timescale=timescale, 
+                              variable=variable)
+    location = LocSelectorArea(area_subset='none', 
+                               cached_area='CA', 
+                               latitude=(32.5, 42), 
+                               longitude=(-125.5, -114), 
+                               name='LocSelectorArea00102')
+    xr_da = _read_from_catalog(selections=selections, location=location)
+    return xr_da
+
+def progressBar(i, tot): 
+    """Display a progress bar inside a for loop 
+    Based on Stack Overflow answer from Mark Rushakoff: https://stackoverflow.com/questions/3002085/how-to-print-out-status-bar-and-percentage
+    
+    Args: 
+        i (int): iteration number
+        tot (int): total number of iterations
+    """
+    j = (i+1) / tot
+    sys.stdout.write('\r[%-20s] %d%% complete' % ('='*int(20*j), 100*j))
+    sys.stdout.flush()  
+    
+
+# Print settings 
+print("Reading in data from AWS for {0} variables: {1}".format(len(variable_list),', '.join(map(str, variable_list))))
+print("Year start: {0}".format(year_start)) 
+print("Year end: {0}".format(year_end))
+print("Timescale: {0}".format(timescale))
+print("Append historical data: {0}".format(append_historical))
+print("Resolution: {0}".format(resolution))
+print("Scenarios: {0}".format(', '.join(map(str,scenarios))))
+
+# Read in each variable individually into an xr.DataArray
+xr_da_list = []
+for i in range(len(variable_list)): 
+    variable = variable_list[i] 
+    xr_da = read_data_for_var(variable=variable, 
+                              year_start=year_start, 
+                              year_end=year_end, 
+                              timescale=timescale, 
+                              append_historical=append_historical, 
+                              resolution=resolution, 
+                              scenarios=scenarios) 
+    xr_da_list.append(xr_da) 
+    progressBar(i, len(variable_list)) # Display progress bar 
+
+
+# Merge to form a single xr.Dataset with several data variables
+xr_ds = xr.merge(xr_da_list)
+
+
+# ----------------- SUBSET DATA TO A SMALLER GEOGRAPHIC REGION -----------------
+
+# To avoid having a huge dataset, we just want to use a geographic subset. 
+# At the time of writing this script (June 20, 2022), the subsetting function built into climakitae was broken. # So, instead we just subset the data using xarray. To do this, we construct a simple lat/lon bounding box of any region we're interested, and filter the data from the dataset using .where()
+# This section constructs a box between San Diego, CA and Joshua Tree, CA
```

Comment:
It's nice to have it be a real place of interest, but it'd be much faster to generate a smaller testing dataset if we avoid using .where, and just .sel(x=slice(arbitrary,arbitrary2),y=slice(arbitrary3,arbitrary4))

Reply:
(Note from discussion with Naomi and Nicole): it's okay to leave as is because this file is only run once